### ISSUE TITLE
jumper to disable microTCA hotplug switch

### ISSUE THREAD

**Issue by sbourdeauducq** [sinara-hw/Sayma_AMC#132]
Since Creotech are shipping boards without front panels (which are delayed), add a jumper to force the µTCA hotplug switch on. Putting tape on it to turn it on is annoying and unreliable.
Same on RTM.

**Comment by marmeladapk** [comment]
This is a one-off issue. Next batch will have panels already. Besides it's easier to force power on in MMC (or even use the front panel button). Jumper isn't needed IMO.

**Comment by sbourdeauducq** [comment]
> Besides it's easier to force power on in MMC

Does this involve IPMI or NATview? If so, it's not easier than a jumper.
Is the front panel button currently wired to force power to both AMC and RTM? This does not sound like such a great idea as users may inadvertently push it while hotplugging boards.

**Comment by kaolpr** [comment]
No, it does not involve IPMI / NATview. What @marmeladapk proposed is modification to AMC MMC, which manages RTM power supply. We can than use AMC reset button to signalize MMC that RTM / AMC is to be switched on/off. Please note that this is **temporary workaround** until panels are produced. Currently, there is very limited number of users (actually, besides Creotech you are the only one), so we believe that exposition on inadvertent actions is minimal.

**Comment by marmeladapk** [comment]
Or you could turn it on and off using mmc console simulating hotswap position to MMC.

**Comment by sbourdeauducq** [comment]
Any update on those front panels?
Also, how do I force power on with the MMC console?

**Comment by marmeladapk** [comment]
@sbourdeauducq We should get all front panels on the last week of October.

> how do I force power on with the MMC console?

Currently you can't, we'd have to add this to MMC console. Is this critical for you now?

**Comment by sbourdeauducq** [comment]
Not critical but replacing the tape on the switches every few days is annoying. I guess I could also solder them short...

**Comment by sbourdeauducq** [comment]
@marmeladapk Any update on the front panels or MMC firmware with that feature?

**Comment by sbourdeauducq** [comment]
> MMC firmware with that feature

Nevermind, I soldered the switches short.

As @jbqubit reminded you, please create a wiki page listing all boards and where one can log reworks made to them.
